### DISCUSSION TITLE
Getting Started with Cloud-native AI using Kubeflow | Glasskube

### DISCUSSION THREAD

**Discussion by giscus** [glasskube/glasskube#1399] [Blog]
# Getting Started with Cloud-native AI using Kubeflow | Glasskube

In this article, I would like to introduce to you Kubeflow: A complete and cloud-native platform that simplifies AI operations.
Join me in setting up Kubeflow on GKE for your organization and get started with cloud-native AI today.

https://glasskube.dev/blog/kubeflow-setup-guide/

**Comment by 6footplus** [comment]
This has been very helpful. Thank you!
